_dasldemos/computational_rank.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Why is your computing environment important?
+main_authors: laderast
+primary_reviewers: vortexing, tfirman
+---
+
+In this page, we talk about reproducible best practices to use software on the FH cluster (`rhino` / `gizmo`). If you're getting started on `rhino` / `gizmo`, this page is for you.
+
+## Learning Objectives
+
+After reading this page, you will be able to
+
+- **Describe** what the dependency nightmare is for software
+- **Utilize** software easymodules on `rhino`/`gizmo` and integrate them into your scripts
```

Comment:
On terminology- "easymodule" isn't a term I've seen for these. There are "easyconfigs" used by "easybuild" which builds the software but what we're deploying is an "Lmod environment module", which is a bit cumbersome.  

Most of our [existing documentation](https://github.com/FredHutch/wiki/blob/2f86403d9e40deec702f69ad98d3bf7a5a7b467d/_scicomputing/compute_environments.md?plain=1#L17) refers to these as "environment modules"

Reply:
Changed wording from Easymodules to Environment Modules